lib/tests/streamlit/script_request_queue_test.py

```diff
@@ -78,6 +78,7 @@ def test_rerun_data_coalescing(self):
         interface.)
         """
         queue = ScriptRequestQueue()
+        widget_mgr = SessionState()
```

Comment:
nit: should we rename this variable `session_state`?

Reply:
I was unsure about it, but probably should. At least shouldn't take any time to replace all.